assets/css/main.scss

```diff
@@ -1,3 +1,38 @@
 body {
-  color: red;
+}
+
+.my-team {
+  & {
```

Comment:
Do you need the & ?

```
.my-team {
  width: 80%;
    margin: 0 auto;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
}
```

would this work?

Reply:
yes, it would work also, my intention is to make all sub-elements appearing same way, same indentation.